src/frontend/hooks/useProjectInvite.ts

```diff
@@ -0,0 +1,33 @@
+import {Invite} from '@mapeo/core/dist/invite-api';
+import {useAcceptInvite, useInvites, useRejectInvite} from './server/invites';
+import {useApi} from '../contexts/ApiContext';
+import {useCallback} from 'react';
+
+export function useProjectInvite() {
+  const invites = useInvites().data;
+  // this will eventually sort invite by date
+  const invite = invites[0];
+  const acceptMutation = useAcceptInvite(invite?.inviteId);
+  const rejectMutation = useRejectInvite(invite?.inviteId);
+  const mapeoApi = useApi();
+
+  const resetState = () => {
+    acceptMutation.reset();
+    rejectMutation.reset();
+  };
+
+  const clearAllInvites = useCallback(() => {
+    invites.forEach(inv => {
+      mapeoApi.invite.reject(inv);
+    });
+  }, [invites, mapeoApi]);
```

Comment:
@EvanHahn this is not ideal as everytime we delete an invite react re-renders. Is it possible to create an api call `mapeoApi.invite.clearAllPendingInvites()`

Reply:
Sure. Would `rejectAllPendingInvites()` work?

Reply:
Is there a world where the user accepts an invite and we do NOT want to clear the pending invites? IF there is, then I think it would make sense for the front end to handle clearing the pending invites.

If we want to always clear the pending invites on accept, then I agree with you andrew, this should be handled by the backend. 

I can imagine that maybe in the future we want to list all pending invites...but we can expose that in the future if we do need that. So for now @EvanHahn lets just clear all invites on accept in the backend and not expose an API for it?